service/frontend/workflow_handler.go

```diff
@@ -5873,3 +5873,110 @@ func (wh *WorkflowHandler) ResetActivity(
 
 	return &workflowservice.ResetActivityResponse{}, nil
 }
+
+func (wh *WorkflowHandler) CreateWorkflowRule(
+	ctx context.Context,
+	request *workflowservice.CreateWorkflowRuleRequest,
+) (_ *workflowservice.CreateWorkflowRuleResponse, retError error) {
+	defer log.CapturePanic(wh.logger, &retError)
+
+	if !wh.config.WorkflowRulesAPIsEnabled(request.GetNamespace()) {
+		return nil, status.Errorf(codes.Unimplemented, "method CreateWorkflowRule not supported")
```

Comment:
I understand you copied it from somewhere else but strong believe is that it should be:
```suggestion
		return nil, serviceerror.NewUnimplemented("method CreateWorkflowRule not supported")
```
And I have a reason for it.

Reply:
which is? Because in this case I will need to change it in other places as well.

Reply:
What will be the error on the caller side if caller is using `CreateRule` function from SDK? It will be `serviceerror.NotImplemented`. The idea of `serviceserror` is that it hides gRPC statuses from our code and allows caller to handle exact same error as returned from handler. This is one of them. General consistency with other errors even in this func, is anther.